local2pod.js

```diff
@@ -0,0 +1,46 @@
+import { SolidNodeClient } from "solid-node-client";
+const client = new SolidNodeClient;
+
+async function local2pod(){
+  try {
+    await client.login();
+  }
+  catch(e){ console.log(e); }
+  for(let file of files){
+    const from = `file:///home/jeff/solid/catalog/${file}`;
+    const to   = `https://solidproject.solidcommunity.net/catalog/${file}`;
```

Comment:
```suggestion
    const to   = `https://solidproject.solidcommunity.net/catalog/${path.basename(file, path.extname(file))}`;
```

this will drop the extension, just need to import `path` first

Reply:
I know how to drop the extension.  The issue is that I do not want the local script to behave differently from the pod script.